contribute-documentation.md

```diff
@@ -39,6 +39,14 @@ The script compiles the Markdown files into static HTML pages and starts a local
 
 Please feel free to ask any questions you have on the developer mailing list.
 
+## Chinese documentation translation
+
+The Flink community is maintaining both English and Chinese documentation. If you want to update or extend the documentation, both English and Chinese documentation should be updated. If you are not familiar with Chinese language, you can open a JIRA with the `chinese-translation` component for Chinese documentation translation and link it with the current JIRA issue. If you are familiar with Chinese language, you are encouraged to update both sides in one pull request.
+
+*NOTE: The Flink community is translating Chinese documentations, some documents may not have been translated yet. If the document you are updating is not translated yet, you can just copy the English changes to the Chinese document.*
```

Comment:
"The Flink community is translating Chinese documentations" 
-> 
"The Flink community is still in the process of translating Chinese documentations"